app/controllers/concerns/aic_asset_delete_behavior.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+module AICAssetDeleteBehavior
+  def delete_asset_and_update_its_collections(curation_concern)
+    curation_concern.in_collection_ids.each do |id|
+      destination_collection = ::Collection.find(id)
+      destination_collection.members.delete(curation_concern)
+      destination_collection.update_index
+    end
```

Comment:
So @awead, you're saying deleting works from collections they belong to doesn't need to happen?